projects/packages/search/src/dashboard/class-initial-state.php

```diff
@@ -80,6 +80,7 @@ public function get_initial_state() {
 				'search'                 => $this->module_control->is_active(),
 				'instant_search_enabled' => $this->module_control->is_instant_search_enabled(),
 			),
+			'features'        => filter_var_array( explode( ',', $_GET['features'] ), FILTER_SANITIZE_STRING ),
```

Comment:
We'll want to fix this since it makes one of the mandatory PHPCS checks fail. 

We could either enforce nonce verification or add in a directive to disable PHPCS for this line:

```suggestion
			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
			'features'        => filter_var_array( explode( ',', $_GET['features'] ), FILTER_SANITIZE_STRING ),
```

Reply:
Thanks @jsnmoon. I was having some issues with PHPCS and PHP 8.1. 

We can only use nonce verification if there's a form to put the nonce into right? 

I'll add the phpcs ignore.